### PR TITLE
BZ1904396: add reference info for the machine approver and the cluster config Operators

### PR DESCRIPTION
**Pull request by chinmayi-chandrasekar**
Applies to 4.6+

Doc preview link: https://deploy-preview-36322--osdocs.netlify.app/openshift-enterprise/latest/operators/operator-reference.html

https://bugzilla.redhat.com/show_bug.cgi?id=1904396

Requires ack from @wabouhamad @abhinavdahiya @elmiko

### CODE REVIEW

File: modules/cluster-config-operator.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+[id="cluster-config-operator_{context}"]
+= Cluster Config Operator
+
+[discrete]
+== Purpose
+
+The Cluster Config Operator performs the following tasks related to `config.openshift.io`:
+
+. Creates CRDs.
```
Comment (sagidlow) [inline modules/cluster-config-operator.adoc:13]: These should be bullets and not a numbered list.

**Review by sagidlow — COMMENTED** [review]
LGTM. One comment regarding bulleted list vs. numbered list.

File: modules/cluster-config-operator.adoc (hunk 1 above)
Comment (chinmayi-chandrasekar) [reply modules/cluster-config-operator.adoc]: Thanks for reviewing. Addressed.

File: modules/cluster-config-operator.adoc [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+//
+// *  operators/operator-reference.adoc
+
+[id="cluster-config-operator_{context}"]
+= Cluster Config Operator
+
+[discrete]
+== Purpose
+
+The Cluster Config Operator performs the following tasks related to `config.openshift.io`:
```
Comment (abhinavdahiya) [inline modules/cluster-config-operator.adoc:11]:
```suggestion
The Cluster Config Operator performs the following tasks related to `config.openshift.io` APIs:
```

File: modules/cluster-config-operator.adoc [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+* Creates CRDs.
+* Renders the initial custom resources.
```
Comment (abhinavdahiya) [inline modules/cluster-config-operator.adoc:14]:
```suggestion
* Renders the initial custom resources during bootstrapping phase of the cluster installation.
```

**Review by abhinavdahiya — COMMENTED** [review]
modules/cluster-config-operator.adoc LGTM

**Review by elmiko — COMMENTED** [review]
the cluster-machine-approver doc looks correct to me
/lgtm

**Pull request merged**